python/tvm/relay/frontend/tensorflow2.py

```diff
@@ -325,12 +475,30 @@ def _convert_operator(self, graph, op_name, node_name, inputs, attrs):
                 sym = _convert_map_common[op_name](inputs, attrs, self._params, self._prelude)
             else:
                 sym = _convert_map_common[op_name](inputs, attrs, self._params, self._module.mod)
+        elif op_name in _convert_map_tf2:
```

Comment:
Should the priority go to using the TF2 converter if an op converter happens to be registered in both TF1 and TF2 convert_map?

Reply:
So far we only have few ops in tf2 converter.

Reply:
The two op lists are exclusive. Tf2 only has the TensorList ops which are not present in Tf1.

Reply:
Add an assert to make sure the ops are exclusive.

Reply:
comments addressed